src/session.rs

```diff
@@ -295,4 +320,16 @@ impl Session {
     pub async fn close(self) -> Result<(), Error> {
         delegate!(self.0, imp, { imp.close().await })
     }
+
+    /// Leak the underlying ssh multiplex master.
+    /// Return (path to control socket, path to ssh multiplex output log)
+    pub fn leak(self) -> (Box<Path>, Option<Box<Path>>) {
+        delegate!(self.0, imp, { imp.leak() })
+    }
+
+    /// Force terminate the remote connection, even if it is created
+    /// by [`Session::resume`] or [`Session::resume_mux`].
+    pub async fn force_terminate(self) -> Result<(), Error> {
```

Comment:
Here, too, I think we can make the API a little clearer. One option is to rely on the same kind of configuration as mentioned above (`close_on_drop(yes: bool)`), where that defaults to no for resumed sessions, but can be set to yes. Alternatively, I think it's reasonable to make an explicit call to `close` _always_ mean `-O exit` (including for resumed sessions), and to _just_ change the default behavior of drop depending on whether the session was resumed or not.

Reply:
Done!